hrui/js/src/hrui.js

```diff
@@ -294,38 +229,14 @@
   function toggleActiveClassOnHoverOnAnyMainMenuItem () {
     var className = 'activetarget';
 
-    $('.menumain:not(.crm-Self_Service_Portal').hover(function () {
+    $('.menumain').not('.crm-Self_Service_Portal').hover(function () {
```

Comment:
Doesn't :not() work? Is it just for readability purposes?

Reply:
please read the PR description, i've explained it there